deploy/charts/README.md

```diff
@@ -0,0 +1,30 @@
+# Helm charts
```

Comment:
nit:  Charts in DC context has a specific meaning. Perhaps rename the directory to deploy/helm_charts/, so a casual reader of the repo isn't confused?

Reply:
Good point, changed to helm_charts.